traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.HTTP.tpl.html

```diff
@@ -655,6 +655,19 @@
                     </div>
                 </div>
 
+                <div class="form-group" ng-class="{'has-error': hasError(deliveryServiceForm.consistentHashRegex), 'has-feedback': hasError(deliveryServiceForm.consistentHashRegex)}">
+                    <label class="control-label col-md-2 col-sm-2 col-xs-12">
+                        <span uib-popover-html="label('consistentHashRegex', 'desc')" popover-trigger="click" popover-placement="top" popover-append-to-body="true" popover-class="popover-class">{{label('consistentHashRegex', 'title')}}</span>
+                    </label>
+                    <div class="col-md-10 col-sm-10 col-xs-12">
+                        <input name="consistentHashRegex" type="text" class="form-control" ng-model="deliveryService.consistentHashRegex" ng-maxlength="1024" autofocus>
+                        <small class="input-error" ng-show="hasPropertyError(deliveryServiceForm.consistentHashRegex, 'maxlength')">Too Long</small>
+                        <small class="input-diff" ng-show="settings.isRequest && open() && deliveryService.consistentHashRegex != dsCurrent.consistentHashRegex">Current Value: [ {{dsCurrent.consistentHashRegex}} ]</small>
```

Comment:
I know you're just copying existing form control markup, but imo this `ng-show` is a little wacky. The thing will never be displayed if this isn't a ds *Request*, which is a known constant on pageload. So imo it should actually be more like:
```html
<small class="input-diff" ng-if="settings.isRequest" ng-show="open() && deliveryService.consistentHashRegex !== dsCurrent.consistentHashRegex>Current Value: [ {{dsCurrent.consistentHashRegex}} ]</small>
```
... it's all about reducing the work done by that **front-end compiler** (:nauseated_face:). Also note the `!==` instead of `!=`, to account for the fact that `==` does type coercion, such that e.g. `0 == "0"` is `true` but `0 === "0"` is `false`. Things get really crazy when you consider more complex objects. We have an ESLint configuration file that would probably warn about this, but I'm not sure if it checks JS expressions in angular attributes.

Reply:
I think that this should be left as is to match the rest of the TP files. you can create an issue if you want it to be addressed in the future

Reply:
+1 on keeping it consistent w/ the rest of the TP files. Fixing all of that would not be in the spirit of this PR.

Reply:
fair enough